### PR TITLE
Add Small Basic

### PR DESCRIPTION
**Pull request by RemiixInc**

### CODE REVIEW

File: aaa.sb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+While ("True")
+  TextWindow.WriteLine("a")
```
Comment (Benricheson101) [inline aaa.sb]: Use `Write` instead of `WriteLine` so it prints on one line

Comment (democat3457) [reply aaa.sb]: clearly you don't know how to make actual suggestions
```suggestion
  TextWindow.Write("a")
```

Comment (democat3457) [reply aaa.sb]: also you can commit to the branch yourself and make the change, especially if it's as small as this

Comment (Benricheson101) [reply aaa.sb]: > clearly you don't know how to make actual suggestions

i didnt realize that was a thing lmfao

Comment (democat3457) [reply aaa.sb]: Screen Shot 2021-03-19 at 10 56 32 PM
now you know

**Pull request merged**